workspaces/orchestrator/plugins/orchestrator/src/api/OrchestratorClient.ts

```diff
@@ -104,10 +111,39 @@ export class OrchestratorClient implements OrchestratorApi {
     const defaultApi = await this.getDefaultAPI();
     const reqConfigOption: AxiosRequestConfig =
       await this.getDefaultReqConfig();
+    // We know the injected ScmIntegrationsApi instance is actually a ScmIntegrationRegistry with .list()
+    const integrations = (this.scmIntegrationsApi as any).list?.();
+    const authTokens: { provider: string; token: string }[] = [];
+    for (const integration of integrations) {
+      const provider = integration.type;
+      const host = integration.config.apiBaseUrl || integration.config.host;
```

Comment:
The default `apiBaseUrl` value is `https://api.github.com/` and when it's using this URL to fetch the token, it fails.
Using the `host` alone make it works fine (so `https://github.com/`).

Is there a reason to use `apiBaseUrl`  instead of `host`?
Or is it a github related behaviour?
cc @mareklibra if you know anything about that